data/human/human missions.txt

```diff
@@ -3767,11 +3767,66 @@ mission "Earth Retirement"
 
 
 
+event "charles donna timer"
+
+
+
+mission "Earth Retirement 2"
+	name "Retired Family to Shangri-La"
+	description "Transport two seniors and their son to <destination>, where they plan to retire. They will pay you <payment>."
+	minor
+	source "Earth"
+	destination "Shangri-La"
+	to offer
+		has "event: charles donna timer"
+		has "outfit: Luxury Accommodations"
+	passengers 3
+	on offer
+		conversation
+			`As you enter the spaceport, you see a familiar sight. The elderly couple Charles and Donna Singer, who you helped move to Earth a while ago, are talking to a young well-dressed man. Donna looks much the same as you remember her, but Charles is sporting a bandage on his head, and one of his arms looks like it's in a sling. When they see you, Donna points excitedly in your direction, and the entire group moves to meet you.`
+			`	"Captain! I never thought we would see you again," Donna begins. "I'd like to introduce you to my son, Abel." She gestures to the young man, who shakes your hand vigorously. Now that you're close, you can see that he is dressed very finely. His suit is made of expensive wool with ebony buttons, his shoes are emblazoned with designer logos, and you can see a jewel-encrusted watch peeking out from underneath the well-tailored cuff of one arm.`
+			`	"It's great to meet you, Captain...?"`
+			choice
+				`	"<first> <last>."`
+				`	"<last>."`
+			`	"Captain <last>," he smiles. "How very nice to meet you indeed. My parents here told me what you did for them. If I may introduce myself more fully, I am Abel Singer, Junior Vice President of Marketing Operations for Syndicated Systems. I want to thank you for going out of your way to help my parents when they needed it most."`
+			choice
+				`	"It was nothing."`
+				`	"What happened to Charles?"`
+					goto injuries
+				`	"How could you leave them to rot...?"`
+					goto rot
+			`	"It was not nothing," Donna blurts out. "You were there for us when nobody else cared. You didn't even accept the agreed upon payment! Our former employers, our friends, even our children..." She trails off.`
+			label rot
+			`	"I've not been the best son," Abel says, staring down at his feet. "My parents are very dear to me, of course. But I was so busy with work. I thought they could take care of themselves. So I left them to their own devices. I didn't even think to wonder about their financial position, and my father here was too proud to say it out loud." He turns to look his father in the eyes. "Still, I should have known better. I'm sorry, dad."`
+			label injuries
+			`	Charles grunts, but it is his wife who speaks up next. "Shortly after we arrived, a gang of local teenagers began terrorizing our retirement community. It was mostly just vandalism, some petty theft, and even some burglaries. Nothing would have happened if Charles had just given up the necklace-"`
```

Comment:
"It was mostly... even" doesn't work here. This might flow better.

```suggestion
			`	Charles grunts, but it is his wife who speaks up next. "Shortly after we arrived, a gang of local teenagers began terrorizing our retirement community. At first it was just vandalism, but eventually it became petty theft, burglary... Nothing would have happened if Charles had just given up the necklace-"`
```